meta-lxatac-software/conf/distro/tacos.conf

```diff
@@ -1,7 +1,7 @@
 DISTRO = "tacos"
 DISTRO_NAME = "TAC OS - The LXA TAC operating system"
 DISTRO_VERSION = "24.04+dev"
```

Comment:
Should this be bumped as well or only when a release is done?

Reply:
It will be bumped once we prepare the next release (and know in which month it is going to happen). Until then we carry the `+dev` suffix, which indicates that the version is built on top of release 24.04.